inlong-common/src/main/java/org/apache/inlong/common/pojo/sort/dataflow/field/format/DateFormatInfo.java

```diff
@@ -126,11 +137,7 @@ public Date deserialize(String text) throws ParseException {
                 return new Date(millis);
             }
             default: {
-                if (simpleDateFormat == null) {
-                    throw new IllegalStateException();
-                }
-
-                java.util.Date jDate = simpleDateFormat.parse(text.trim());
+                java.util.Date jDate = get().parse(text.trim());
```

Comment:
If it is invalid date string, this will throw exception, it is better to return null or default value.

Reply:
It should throw ParseException when the input is an invalid string, the invoker should handle this exception.